addon/components/marker-cluster-layer.js

```diff
@@ -52,6 +52,18 @@ export default class MarkerClusterLayer extends BaseLayer {
   componentsToYield = [
     ...this.componentsToYield,
     { name: 'marker-layer', as: 'marker' },
+    { name: 'tile-layer', as: 'tile' },
```

Comment:
@bzf what layers can be added to a marker cluster?

I'm pretty sure you can't add tile-layers to marker clusters, right?

Reply:
TBH, I don't know. I think I just moved over everything from https://github.com/miguelcobain/ember-leaflet/blob/32d13608d88f576da934f0e2f4217353fafeeb97/addon/templates/leaflet-map.hbs, which seems pretty wrong.

I'm only using the `circle-marker` one in my scenario - maybe it's smarter to only add the one I know for sure works?